assets/src/components/mapMarkers.tsx

```diff
@@ -281,8 +285,31 @@ export const RouteStopMarkers = ({
             key={stop.id}
             stop={stop}
             direction={direction}
-            includeStopCard={includeStopCard}
+            includeStopCard={includeStopCard && !streetViewActive}
```

Comment:
This was on purpose for two reasons
- It looked really weird to not have any interactivity when in street view mode
- it was less code to write

I do have this question in my design/product review ticket, so this will be confirmed with design before merge.

Reply:
Kathleen approves, so this is in!